ipatests/test_webui/ui_driver.py

```diff
@@ -1875,3 +1875,132 @@ def assert_action_list_action(self, action, visible=True, enabled=True,
             is_enabled = not self.has_class(li, 'disabled')
             assert is_enabled == enabled, ('Invalid enabled state of action item %s. '
                                            'Expected: %s') % (action, str(visible))
+
+    def assert_field_validation(self, expect_error, parent=None):
+        """
+        Assert for error in field validation
+        """
+
+        if not parent:
+            parent = self.get_form()
+
+        req_field_css = '.help-block[name="error_link"]'
+
+        res = self.find(req_field_css, By.CSS_SELECTOR, context=parent)
+        assert expect_error in res.text, 'No error found'
```

Comment:
Instead of 'No error found' I would add something like:

'Expected error: {} not found'.format(expect_error)

Reply:
You may also want do add another assert before to check if we even have some element. So we do not fail on: 'NoneType' object has no attribute 'text'. Or do strict=True ? But that would raise exception anyway, @pvoborni what would be right approach here?